### PR TITLE
[Search] Add search feature and link to details

### PR DESCRIPTION
**Pull request by Naarco**
- move search bar beneath 'my series' (we don't want the series to move depending on search results length)

### CODE REVIEW

File: seriespy/static/js/home.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        // only consider successful requests when they are done
+        if (xmlHttp.readyState == 4 && xmlHttp.status == 200) {
+            var res = JSON.parse(xmlHttp.responseText);
+            // iterate over the results to show names and link to the detail page
+            for (var i = 0; i < res.length; i++) {
+                var showName = res[i]["show"]["name"];
+                var listItem = document.createElement('li');
+                listItem.className = "search-result-item";
+                listItem.dataset.link = "/details/" + res[i]["show"]["id"];
+                //var link = document.createElement("a");
```
Comment (Succo) [inline seriespy/static/js/home.js]: Virer les comments?

File: seriespy/static/js/home.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                var image = document.createElement("img");
+                var span = document.createElement("span");
+                if (res[i]["show"]["image"]) {
+                    image.src = res[i]["show"]["image"]["medium"];
+                    image.className="right circle";
+                    listItem.appendChild(image);
+                }
+                span.className="highlight";
+                //link.href = "/details/" + res[i]["show"]["id"];
```
Comment (Succo) [inline seriespy/static/js/home.js]: idem?

File: seriespy/static/js/home.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                span.innerHTML = showName;
+                listItem.appendChild(span);
+                //listItem.appendChild(link);
```
Comment (Succo) [inline seriespy/static/js/home.js]: idem?

**Comment by MarwanEB** [comment]
:shipit:

**Pull request merged**